docker-jans-fido2/scripts/upgrade.py

```diff
@@ -22,8 +22,6 @@ def _transform_fido2_dynamic_config(conf):
 
     # add missing top-level config (if not exist)
     for k, v in [
-        ("errorReasonEnabled", False),
-        ("sessionIdPersistInCache", False),
     ]:
```

Comment:
This should be removed entirely since the array is empty. The subsequent if clause should be removed too. 
cc @iromli